src/main/resources/assets/jei/lang/zh_tw.json

```diff
@@ -43,81 +43,85 @@
+  "config.jei.search.resourceIdSearchMode.comment": "依 Resource ID 搜尋（前綴：&）",
+  "config.jei.advanced": "進階設定",
+  "config.jei.advanced.comment": "用於變更 JEI 功能的進階設定",
   "config.jei.advanced.itemBlacklist": "物品黑名單",
-  "config.jei.advanced.itemBlacklist.comment": "黑名單中的物品將不會顯示於JEI物品列表，格式：modId[:name[:meta]]。使用編輯模式可自動增加或删除這這裏的條目",
+  "config.jei.advanced.itemBlacklist.comment": "黑名單中的物品將不會顯示於 JEI 物品列表，格式：modId[:name[:meta]]。使用編輯模式可自動增加或刪除這這裡的條目",
   "config.jei.advanced.colorSearchEnabled": "啟用顏色搜尋",
   "config.jei.advanced.colorSearchEnabled.comment": "依顏色搜尋物品，並在可以這樣搜尋的物品提示中加以注明",
   "config.jei.advanced.debugModeEnabled": "測試模式",
-  "config.jei.advanced.debugModeEnabled.comment": "僅對JEI開發者有用，啟用後會增加測試物品及一些測試配方",
+  "config.jei.advanced.debugModeEnabled.comment": "僅對 JEI 開發者有用，啟用後會增加測試物品及一些測試配方",
   "config.jei.advanced.centerSearchBarEnabled": "置中搜尋欄",
   "config.jei.advanced.centerSearchBarEnabled.comment": "啟用後，搜尋欄會被移動至底部中間的位置",
   "config.jei.advanced.modNameFormat": "模組名稱格式",
-  "config.jei.advanced.modNameFormat.comment": "定義JEI介面的資訊提示中的模組名稱格式，留空以停用",
+  "config.jei.advanced.modNameFormat.comment": "定義 JEI 介面的資訊提示中的模組名稱格式，留空以停用",
   "config.jei.advanced.maxColumns": "最大寬度",
   "config.jei.advanced.maxColumns.comment": "物品列表的最大寬度",
   "config.jei.advanced.maxRecipeGuiHeight": "最大配方列表介面高度",
   "config.jei.advanced.maxRecipeGuiHeight.comment": "配方列表的最大高度",
   "config.jei.advanced.giveMode": "物品給予方法",
-  "config.jei.advanced.giveMode.comment": "選擇JEI應該將物品直接提供給物品欄（inventory），或者是使用鼠標進行物品提取（mouse_pickup）。",
+  "config.jei.advanced.giveMode.comment": "選擇「JEI 應該將物品直接放到物品欄」，或者是「使用滑鼠進行物品提取」。",
 
   "_comment": "Hide Ingredients Mode",
-  "gui.jei.editMode.description": "JEI物品列表編輯模式：",
-  "gui.jei.editMode.description.hide": "按住%s並點擊以隱藏",
-  "gui.jei.editMode.description.hide.wild": "按住%s並右擊以隱藏相同ID的物品",
+  "gui.jei.editMode.description": "JEI 物品列表編輯模式：",
+  "gui.jei.editMode.description.hide": "按住 %s 並點擊以隱藏",
+  "gui.jei.editMode.description.hide.wild": "按住 %s 並點擊右鍵以隱藏相同 ID 的物品",
 
   "_comment": "PC/Linux Control Key",
   "key.jei.ctrl": "CTRL",
 
   "_comment": "Mac Command Key",
   "key.jei.ctrl.mac": "CMD",
 
-  "_comment": "Recipe Categories 合成類別",
+  "_comment": "配方種類",
   "gui.jei.category.craftingTable": "合成",
-  "gui.jei.category.smelting": "冶煉",
-  "gui.jei.category.smelting.experience": "%s經驗",
+  "gui.jei.category.smelting": "熔煉",
+  "gui.jei.category.smoking": "燻製",
+  "gui.jei.category.blasting": "熔煉",
+  "gui.jei.category.campfire": "營火野炊",
+  "gui.jei.category.smelting.experience": "%s 經驗",
   "gui.jei.category.fuel": "燃料",
-  "gui.jei.category.fuel.smeltCount.single": "燒煉1個物品",
-  "gui.jei.category.fuel.smeltCount": "燒煉%s個物品",
+  "gui.jei.category.fuel.smeltCount.single": "可熔煉 1 個物品",
+  "gui.jei.category.fuel.smeltCount": "可熔煉 %s 個物品",
   "gui.jei.category.brewing": "釀造",
-  "gui.jei.category.brewing.steps": "需要步驟：%s",
-  "gui.jei.category.itemInformation": "描述",
+  "gui.jei.category.brewing.steps": "步驟： %s",
```

Comment:
Is it necessary to have one whitespace after `：`?

Reply:
That was a mistake. Will be removed.